config/locales/models/user.en.yml

```diff
@@ -5,3 +5,9 @@ en:
     attributes:
       user:
         full_name:
+
+  helpers:
```

Comment:
@KirillKayumov let's move this section to separate file `config/locales/helpers.en.yml`